api/autopilot.go

```diff
@@ -54,19 +51,33 @@ type (
 
 	// HostsConfig contains all hosts settings used in the autopilot.
 	HostsConfig struct {
-		AllowRedundantIPs          bool                        `json:"allowRedundantIPs"`
-		MaxDowntimeHours           uint64                      `json:"maxDowntimeHours"`
-		MinProtocolVersion         string                      `json:"minProtocolVersion"`
-		MaxConsecutiveScanFailures uint64                      `json:"maxConsecutiveScanFailures"`
-		ScoreOverrides             map[types.PublicKey]float64 `json:"scoreOverrides"`
+		AllowRedundantIPs          bool   `json:"allowRedundantIPs"`
+		MaxConsecutiveScanFailures uint64 `json:"maxConsecutiveScanFailures"`
+		MaxDowntimeHours           uint64 `json:"maxDowntimeHours"`
+		MinProtocolVersion         string `json:"minProtocolVersion"`
 	}
 )
 
-// EndHeight of a contract formed using the AutopilotConfig given the current
-// period.
-func (ap *Autopilot) EndHeight() uint64 {
-	return ap.CurrentPeriod + ap.Config.Contracts.Period + ap.Config.Contracts.RenewWindow
-}
+var (
+	DefaultAutopilotConfig = AutopilotConfig{
+		Enabled: false,
+		Contracts: ContractsConfig{
+			Amount:      50,
+			Period:      144 * 7 * 6,
+			RenewWindow: 144 * 7 * 2,
+			Download:    1e12, // 1 TB
+			Upload:      1e12, // 1 TB
+			Storage:     4e12, // 4 TB
```

Comment:
Similar conversation: since we now have an initial contract funding instead of an allowance, is this necessary? Calculating this ahead of time is adds unnecessary cognitive overhead.

Reply:
We could get rid of the remaining storage score and simplify the collateral score and instead just check whether a host is already full / out of collateral before forming a contract without giving them a score.